Dockerfile

```diff
@@ -64,7 +64,7 @@ RUN chmod +x /app/scripts/*.sh \
   && /app/scripts/install_build_dependencies.sh \
   && /app/scripts/install_runtime_dependencies.sh
 
-COPY --from=viktorstrate/dependencies /artifacts.tar.gz /dependencies/
+COPY --from=photoview/dependencies /artifacts.tar.gz /dependencies/
```

Comment:
It should be `ghcr.io/photoview/dependencies`. (or `ghcr.io/photoview/photoview/dependencies` depending on how the workflow works)

Reply:
The `dependencies` imege is always pushed to the new Docker org registry, so there is no need to put a condition here. Anyway, it should be fine to refer to the default location here even if it is a PR or local branch in a forked repo.